### PR TITLE
feat(common/chains): add garnet

### PR DESCRIPTION
**Pull request by qbzzt**
## Description

Add the garnet network

## Testing

None, I don't know how to release packages to verify, and it's a tiny change.

## Additional Information

Your ENS/address: qbzzt.eth

## Summary by CodeRabbit

- **New Features**
	- Introduced a new preset configuration named `garnet`, enhancing the application's functionality.
	- Added a logging configuration and hardfork versioning to the `garnet` preset.

- **Documentation**
	- Updated exports to include the new `garnet` module for easier access within the application.
	- Added comments to guide developers on updating documentation links for new chains.

### CODE REVIEW

File: packages/common/src/presets/garnet.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { garnet as _garnet } from 'viem/chains'
+import { createCommon } from '../createCommon.js'
+
+export const redstone = createCommon({
```
Comment (roninjin10) [inline packages/common/src/presets/garnet.js:4]: Typo in name

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Need to run pnpm changeset and do a minor release

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Can you add another changeset for all packages patch for updating viem

**Comment by qbzzt** [comment]
> Can you add another changeset for all packages patch for updating viem

All of them, or just those that have viem as a dependency?

**Comment by roninjin10** [comment]
Either works I usually just do all of them to be safe since it’s in prerelease mode anyways

**Comment by qbzzt** [comment]
> Either works I usually just do all of them to be safe since it’s in prerelease mode anyways

That's what I did.

**Comment by qbzzt** [comment]
Can you rerun https://github.com/evmts/tevm-monorepo/actions/runs/10874785545/job/30174220169?pr=1434 ? It failed due to a timeout.

**Comment by roninjin10** [comment]
Closing because this pr will be supersceded by https://github.com/evmts/tevm-monorepo/pull/1441 which will generate all these chain files from viem. Viem will also be updated to latest version in an upcoming pr that updates every package to latest

**Pull request closed without merging**